plugins/text_line_dataset.cc

```diff
@@ -1,297 +0,0 @@
-// This file is part of MLDB. Copyright 2015 Datacratic. All rights reserved.
-
-/** text_line_dataset.cc
```

Comment:
heads-up @mailletf this dataset is going away, so you will have to update the Sift notebooks if you rely on it. The replacement is text.csv.tabular with delimiter=quotechar=''.

Note #2: the text.csv.tabular dataset is queued up for being split up into a procedure and a dataset, so there's another change a'comin'.